basic/02c_feedback_loading_log.py

```diff
@@ -5,9 +5,13 @@
 folder = './logs/'
 name = 'exampleLog.hebilog'
 
-log = hebi.util.load_log('{}/{}'.format(logFileFolder, logFileName))
+log = hebi.util.load_log('{}/{}'.format(folder, name))
 
 # Plot using some handy helper functions
 hebi.util.plot_logs(log, 'position', figure_spec=101)
+
+log = hebi.util.load_log('{}/{}'.format(folder, name))
 hebi.util.plot_logs(log, 'velocity', figure_spec=102)
```

Comment:
You can call `plot_logs` with a string object too, so something like:

```py
hebi.util.plot_logs('{}/{}'.format(folder, name), 'velocity', figure_spec=102)
```

but this is sufficient for now.